scripts/format_translations.py

```diff
@@ -201,18 +195,19 @@ def main():
                 sys.exit()
 
     with open(args.filename, 'rb') as csvfile:
-        reader = csv.reader(csvfile)
-        for row in reader:
-            keys = row[0].split('.')
-            merge(data, format_dict(keys, row[1].strip(" ")))
+            reader = csv.reader(csvfile)
+            for row in reader:
+                keys = row[0].split('.')
+                merge(data, format_dict(keys, row[1].strip(" ")))
+
     with open(out_fn, 'w') as f:
-        data.update(numbers)
-        json.dump(data, f, indent=4, sort_keys=True, ensure_ascii=False)
+            data.update(numbers)
```

Comment:
General note: indentation in python is typically 4 char for a new block, not 8.

https://www.python.org/dev/peps/pep-0008/#indentation